x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action_dry_run.ts

```diff
@@ -296,7 +296,7 @@ export default ({ getService }: FtrProviderContext): void => {
       });
     });
 
-    describe('schedule manual rule run action', () => {
```

Comment:
Please help me understand. I see that a `should return all existing and enabled rules as succeeded` test within this describe is failing in [PR pipeline](https://buildkite.com/elastic/kibana-pull-request/builds/221447#0190b6cf-8731-4124-b623-a5a9064b9031). Shouldn't it be skipped since its parent `describe` has `@skipInServerless` tag now?

Reply:
That's an old build, latest (from couple days ago) is here - https://buildkite.com/elastic/kibana-pull-request/builds/221536
Weird, Github didn't update the build history. Anyway, let me run it again.

Reply:
Oh, yeah, GitHub-BuildKite integration has been misbehaving recently. I see that it's not failing in the recent build. I have added it to the [ticket](https://github.com/elastic/kibana/issues/188673) to fix.